tools/publisher/src/subcommand/generate_version_manifest.rs

```diff
@@ -143,14 +143,14 @@ fn find_released_versions(
             let unrecent_version = parse_version(crate_name, &unrecent_version.version)?;
             if unrecent_version != recent_version {
                 // Sanity check: version numbers shouldn't decrease
-                if unrecent_version > recent_version {
-                    bail!(
-                        "Version number for `{}` decreased between releases (from `{}` to `{}`)",
-                        crate_name,
-                        unrecent_version,
-                        recent_version
-                    );
-                }
+                // if unrecent_version > recent_version {
+                //     bail!(
+                //         "Version number for `{}` decreased between releases (from `{}` to `{}`)",
+                //         crate_name,
+                //         unrecent_version,
+                //         recent_version
+                //     );
+                // }
```

Comment:
I had to comment this out since we are going from `0.54.1` to `0.0.0-smithy-rs-head` on `main` - the version will only be set on release branches in `gradle.properties`.
If my understanding is correct, we can uncomment this (and the related test) after this PR has been merged.
@jdisanti @rcoh @Velfi @ysaito1001

Reply:
You can just delete it.

Reply:
Although, I think we might have another problem... The SDK release process always uses the latest `versions.toml` in the `aws-sdk-rust` repo root. If we want to do patch releases of the SDK, I think we'll have to do some modifications to that release process to have it use branches as well.

Reply:
Yes, we'll need to do some changes there to be able to backport fixes for SDK crates.
I don't think changes will be necessary in order to do patch releases if they are the latest release (in the semver sense)?

Reply:
Removed in [f8de002](https://github.com/awslabs/smithy-rs/pull/2253/commits/f8de00227858cef87c63cdb1603b92d755410444)